back/app/controllers/web_api/v1/comments_controller.rb

```diff
@@ -101,7 +101,7 @@ def index_xlsx
       service = XlsxService.new
       xlsx = case @post_type
       when 'Idea'
-        service.generate_idea_comments_xlsx @comments, view_private_attributes: Pundit.policy!(current_user, User).view_private_attributes?
```

Comment:
Is the view_private_attributes? method of the policy still used anywhere? If not, I think we should remove it (or set its value to always true).

Reply:
I liked your idea of leaving the functionality, for now, in case we get some customer push back and end up reverting all or some of these changes.

So, probably best is to make the default value of `view_private_attributes: true` in the `_generate_xlsx` method params, and then just call the methods from the controllers without needing to pass the param.

Reply:
The view_private_attributes? in the User policy is still in use in some serializers, and some other (non XLSX) controller actions.

So, on balance, I think the approach I take here probably makes most sense. Leave the functionality in place, in case we need to use it again soon, but explicitly skip using it in the method calls (so it's really clear we always get private attributes from the XLSX actions).

Reply:
I'll merge this as it is, but I'll schedule a quick meeting with you to consider what to do, if anything, about the redundant code it leaves, etc.

It's simple enough for me to change, and as long as we ensure we aren't permitting anything we shouldn't (I'm confident we're not), then we are really only talking about what constitutes the best coding approach.